boot/zephyr/include/nrf53_cpunet_ctl.h

```diff
@@ -0,0 +1,28 @@
+
+/*
+ * Copyright (c) 2020 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef H_NRF53_CPUNET_CTL_
```

Comment:
Is this the standard way of doing header guards in NCS?

Reply:
fixed